internal/telemetry/telemetry.go

```diff
@@ -55,21 +55,20 @@ func (c *client) configChange(configuration []Configuration) {
 	}
 }
 
-// productChange enqueues an app-product-change event that signals a product has been turned on/off.
-// Must be called with c.mu locked.
-// A productChange message with enabled=true indicates that a certain product has been used for this application.
-// A productChange message with enabled=false indicates this product is inactive/disabled for an application.
-func (c *client) productChange(namespace Namespace, enabled bool) {
+// productOn enqueues an app-product-change event that signals a product has been turned on.
+// Must be called with c.mu locked. An app-product-change event with enabled=true indicates
+// that a certain product has been used for this application.
+func (c *client) productOn(namespace Namespace) {
```

Comment:
```suggestion
func (c *client) productEnabled(namespace Namespace) {
```

slight preference for `productEnabled` since that's what it's called in instrumentation telemetry-speak
Plus when I initially saw it, I was like "wait, what's the product on top of?" and got confused 😆